cli/tests/integrations/test_task.py

```diff
@@ -184,6 +184,9 @@ def test_log_two_tasks():
 
     lines = stdout.decode('utf-8').split('\n')
     assert len(lines) == 19
+    assert len(
+        [line for line in lines if line.startswith('===> task:test-app')]
+    ) == 2
```

Comment:
This is a small improvement to the existing test. When logging more than one task the CLI prepends "===> task:{task-name} ..." before the logs of each task.